tapdb/sqlc/migrations/000012_multiverse_tree.down.sql

```diff
@@ -0,0 +1,3 @@
+DROP INDEX IF EXISTS multiverse_leaves_unique;
```

Comment:
I don't see this index ever constructed?

Reply:
No, you're right, it doesn't exist. Left over from when I tried something.